Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env groovy
 common {
   slackChannel = '#connect-warn'
-  upstreamProjects = 'confluentinc/kafka-connect-storage-common'
+  upstreamProjects = ['confluentinc/kafka-connect-storage-common','confluentinc/schema-registry','confluentinc/common']
```

Comment:
Aren't the dependencies transient?
Meaning, if `confluentinc/kafka-connect-storage-common` depends on `confluentinc/schema-registry` and `confluentinc/common` doesn't suffice to add only the former?

Reply:
Yes, this is correct, and the script does no dependency resolution to determine the minimum set of upstream projects.
I'll close this PR and disable future updates to this branch.